library/src/main/java/com/rtchagas/pingplacepicker/repository/googlemaps/GoogleMapsRepository.kt

```diff
@@ -122,10 +131,127 @@ class GoogleMapsRepository constructor(
                     if (("OK" == result.status) && result.results.isNotEmpty()) {
                         return@flatMap getPlaceById(result.results[0].placeId)
                     }
-                    return@flatMap Single.just(null)
+                    return@flatMap Single.just(PlaceFromCoordinates(location.latitude, location.longitude))
                 }
     }
 
+    /**
+     * Place without any additional info. Just latitude and longitude.
+     */
+    private class PlaceFromCoordinates(private val latitude: Double, private val longitude: Double) : Place() {
```

Comment:
Please put this class into a separate file in googlemaps/ package.

Reply:
OK